### DISCUSSION TITLE
Octolamp build

### DISCUSSION THREAD

**Discussion by metcalfc** [martinwoodward/octolamp#6] [Show and tell]
![IMG_3923](https://user-images.githubusercontent.com/31925/215353537-a9b39276-daf5-4e6c-bb3a-37609b197ac5.jpg)

Prusa i3 MK3S+
Prusament PLA Prusa Galaxy Black (refill roll)
Protopasta Matte White

Stock 0.30mm Draft profile
15% infill

Super easy print. 

When I checked the LEDs I bought I noticed I was going to have to swap the ground and data pins. You can check by looking at your PCBs. I've highlighted the pin labels in red. The LEDs were 5v, Data, Ground. The ESP was 5v, Ground, Data. I could have snipped the connector at soldered the wires directly but it was just as easy to pop the two out and switch them. I had right angled headers so I used them to keep the profile down.

![InkedIMG_3928](https://user-images.githubusercontent.com/31925/215353570-9c176ad0-985c-48a2-a63e-3e25be7f07db.jpg)

**Comment by Likqez** [comment]
Hey @metcalfc, how did u slice the models? When i try to slice the `black_outer` part, it tells me that a toolpath outside the print area was detected.

**Reply by Likqez** [reply]
For all folks wondering, the error occured because the slicer placed an loop around the model. Disabling this, resulted in a working slice.

**Reply by martinwoodward** [reply]
Thanks for getting back to us.
